services/graphql-server/src/graphql/definitions/website/section.js

```diff
@@ -210,6 +227,8 @@ input CreateWebsiteSectionMutationInput {
   site: ObjectID!
   parent: Int
   logo: ObjectID
+  relatedSectionIds: [Int]
```

Comment:
All related IDs should be defined as `[Int!]!` (and then ensure all resolvers will always return an array)

Reply:
This is the input, and not required for creation/update. I will check the type definition though